doc/sphinx/conf.py

```diff
@@ -80,7 +80,7 @@
     "use_repository_button": True,
     "use_edit_page_button": True,
     "use_issues_button": True,
-    "extra_navbar": "<p><img src='/_static/images/cig_logo_dots.png' alt=\"CIG Logo\" height=\"80px\"  style=\"padding: 5px;\"/></p>",
+    "extra_navbar": "<p><img src=\"{{ pathto('_static/' + 'images/cig_logo_dots.png', 1) }}\" alt=\"CIG Logo\" height=\"80px\"  style=\"padding: 5px;\"/></p>",
```

Comment:
Why do you split the string here before combining?
```suggestion
    "extra_navbar": "<p><img src=\"{{ pathto('_static/images/cig_logo_dots.png', 1) }}\" alt=\"CIG Logo\" height=\"80px\"  style=\"padding: 5px;\"/></p>",
```

Reply:
I am just desperately trying to get this to work in any form :-) ... it did work when this was a separate template html (https://github.com/geodynamics/aspect/pull/4735/files#diff-415f2d804b45c8f2c32a3536b67c400686f9800ebe85b1320fd32c116f7582afL2), but does not seem to work as part of this "extra_navbar" element. I think I am down to using an absolute html path to make it work on all pages.